script.js

```diff
@@ -0,0 +1,48 @@
+const btn = document.querySelector('.hamburger');
+
+const navbar = document.querySelector('nav');
+
+const navlink = document.querySelectorAll('.nav-items a');
+const hamburgerImage = document.querySelector('.hamburger-img');
+
+function openMenu() {
+  if (navbar.style.height === 'initial') {
+    navbar.style.height = '100%';
+  } else {
+    navbar.style.height = 'initial';
+  }
+  if (navbar.style.backgroundColor === 'transparent') {
+    navbar.style.backgroundColor = '#396df2';
+  } else {
+    navbar.style.backgroundColor = 'transparent';
+  }
+
+  for (let i = 0; i < navlink.length; i += 1) {
+    if (navlink[i].style.display === 'none') {
+      navlink[i].style.display = 'block';
+    } else {
+      navlink[i].style.display = 'none';
+    }
+  }
+
+  if (hamburgerImage.src === 'img/enabled.png') {
+    hamburgerImage.src = 'img/Toolbar.png';
+  } else {
+    hamburgerImage.src = 'img/toolbar.png';
+  }
+}
+
```

Comment:
- [ ] Please the functionality of your mobile menu is not working, when I click on the hamburger, I don't see any mobile menu. you can check that in the video below

[Screencast from 12-20-2022 09:06:02 PM.webm](https://user-images.githubusercontent.com/92303891/208768624-da37dd2b-1209-44d6-9577-571771aa22a0.webm)